modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -85,7 +85,7 @@ protected function register_controls() {
 		$end = is_rtl() ? 'left' : 'right';
 		$tooltip_start = is_rtl() ? esc_html__( 'Right', 'elementor' ) : esc_html__( 'Left', 'elementor' );
 		$tooltip_end = is_rtl() ? esc_html__( 'Left', 'elementor' ) : esc_html__( 'Right', 'elementor' );
-		$nested_tabs_heading_selector_class = '{{WRAPPER}} > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading';
+		$nested_tabs_heading_selector_class = ':is( {{WRAPPER}} > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading, {{WRAPPER}} > .elementor-widget-container > .e-n-tabs > .e-n-tabs-content )';
```

Comment:
Could you add a `>` charachter after the ` .e-n-tabs-content`? If we do that, we don't have to add them at the selectors below.

Reply:
That was my fist try :)
It didn't work, so I added the direct sign to after each and every use of the `nested_tabs_heading_selector_class` variable.

Reply:
Can we try it again? I believe it should work.

Reply:
We tried together :)
We learned that the specificity sign can't the last element of an `:is` function.
Please resolve.